docs/upgprade-check/upgrade-check.rst

```diff
@@ -0,0 +1,161 @@
+
+    Changes in import paths of hooks, operators, sensors and others
+    ---------------------------------------------------------------
+    Many hooks, operators and other classes has been renamed and moved. Those changes were part of unifying names and imports paths as described in AIP-21.
+    The contrib folder has been replaced by providers directory and packages:
+    https://github.com/apache/airflow#backport-packages
+
+    Problems:
+
+    1.  Using airflow.operators.python_operator.PythonOperator will be replaced by airflow.operators.python.PythonOperator. Affected file:
+
+
+The following sections describe what is being done and how to apply the recommendations shown above.
+Please note that the above results shown are only a partial set, where only the first
+two of the seven problems identified are shown in the section above. In reality,
+all the problems are shown on the screen.
+
+
+Understanding what is being checked
+'''''''''''''''''''''''''''''''''''
+
+The Upgrade Check checks the configuration data from airflow.cfg, the meta data from the Airflow
+database, as well as the DAGs which have been set up in the current Airflow environment.
+
+Using the above results as an example, there are two specific problems which have
+been identified.
+
+The first problem is identified in the configuration file airflow.cfg where the current configuration
+option for the fernet_key is no longer acceptable and needs to be changed. This is because as of
+Airflow 2.0, the fernet_key cannot be left empty, but needs to have a defined value. Examining the
+problematic airflow.cfg and searching for the fernet_key entries would show the following:
+
+.. code-block:: bash
+    fernet_key =
+
+The second problem was identified in one of the DAGs. In this case, this import
+statement for the PythonOperator needs to be changed, since the location is different
+in Airflow 2.0. Examining the DAG file would probably show the following:
+
+.. code-block:: bash
+    from airflow.operators.python_operator import PythonOperator
+
+We will discuss how to fix these and make them compatible with Airflow 2.0 in the next
+section.
+
+
+Applying the Recommendations
+''''''''''''''''''''''''''''
+
+In most cases, the Recommendations result section of the Upgrade check contains
+enough information to make the change.
+
+For the first problem identified above with respect to the fernet_key, the solution is
+to enter a valid value in the Airflow Configuration file airflow.cfg for the fernet_key.
+
+For the second problem, looking at the source of the DAG file and changing the import
+statement for the Python Operator to be as follows will make this DAG work in Airflow 1.10.14
+as well as make it compatible for Airflow 2.0.
+
+.. code-block:: python
+    from airflow.operators import PythonOperator
```

Comment:
This does not work with 1.10.14 does it @vikramkoka ?

Reply:
Actually, this does seem to work in 1.10.14rc1

Reply:
I will retest with 1.10.14rc4 and 2.0beta3 in just a bit to make sure.

Reply:
Drats! This does work in 1.10.14rc4, but it does NOT work in 2.0.0b3. 
I will fix it in the doc, but not right now, I don't know of a way where it works in both.

Reply:
See: https://github.com/apache/airflow/commit/0e5eee83b14b2a57345370b14e91404d518f0bf4#diff-4169e001fd63ad284011b12a578f74ae09ebeda1c5318256c0d7a97633c738c8